docs/allclasses-frame.html

```diff
@@ -1,18 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
Sorry for making the mess; should've just put on `gh-pages` manually earlier. Want to purge these from the history as unnecessary?